packages/react-components/src/components/chart/trendCursor/getTrendCursor/components/markers/index.ts

```diff
@@ -21,7 +21,7 @@ const addTCMarker = ({
   return {
     id: `circle-${index}-${uId}`,
     type: 'circle',
-    ignore: false,
+    ignore: (series[index] as LineSeriesOption).lineStyle?.opacity === 0 ? true : false,
```

Comment:
when an `if` statement returns `true` else `false`
its a sign we dont need the if! 

this can be simplified to `(series[index] as LineSeriesOption).lineStyle?.opacity === 0`